### ISSUE TITLE
Test download of sample dataset on keeper via the browser

### ISSUE THREAD

**Issue by pravsripad** [Cogitate-consortium/cogitate-msp1#40]
Sample datasets have been uploaded on keeper to help test / demo the code as part of MSP1. Since there are many files and very big files included in the dataset, the data should be downloaded via browser once to check if it works. 

@nbonacchi I think you were already planning to do this.

**Comment by nbonacchi** [comment]
Yeah, keeper will not zip and download the files it complains they are too big. I'll try zipping and uploading per modality to see if I can get a DL link to share in the repo.
I downloaded all the sample data (via sync) and am going to try and run the code for the other modalities (non meeg) beh_et is empty though.
